prow/cmd/gerrit/main.go

```diff
@@ -97,7 +184,20 @@ func main() {
 		logrus.WithError(err).Fatal("Error getting kube client.")
 	}
 
-	c, err := adapter.NewController(o.lastSyncFallback, o.cookiefilePath, o.projects, kc, cfg)
+	ctx := context.Background() // TODO(fejta): use something better
+	op, err := io.NewOpener(ctx, o.gcsCredentialsFile)
+	if err != nil {
+		logrus.WithError(err).Fatal("Error creating opener")
+	}
+	st := syncTime{
+		path:   o.lastSyncFallback,
+		ctx:    ctx,
+		opener: op,
+	}
+	if err := st.init(); err != nil {
```

Comment:
Or init will fail if no `gcsCredentialsFile` is provided? Seems we are not relying on the default credential (no `gcsCredentialsFile` means no gcs client), which should be ok

Reply:
If `gcsCredentialsFile == ""` then we create a new storage client without passing the `WithCredentialsFile` option https://godoc.org/google.golang.org/api/option#WithCredentialsFile, which will probably use the default service account when on GKE